bot/utils/blurple_formatter.py

```diff
@@ -172,7 +180,68 @@ def indent(nodes: list[ast.AST], n: int) -> Generator[str, None, None]:
     yield n * " " + space().join(curr_chunk)
 
 
-def unparse(node: ast.AST, nl_able: bool = False) -> str:
+def _str_literal_helper(string, *, quote_types=_ALL_QUOTES, escape_special_whitespace=False):
+    """
+    Helper for writing string literals, minimizing escapes.
+    Returns the tuple (string literal to write, possible quote types).
+    Equivalent behaviour to ast._Unparser._str_literal_helper.
+    """
+    def escape_char(c):
+        # \n and \t are non-printable, but we only escape them if
+        # escape_special_whitespace is True
+        if not escape_special_whitespace and c in "\n\t":
```

Comment:
The comment, and the code implementation, contradict each other here.

Reply:
No, they don't after close inspection. This code was just copied from `ast._Unparser`.